lib/dataspec/src/histogram.rs

```diff
@@ -14,13 +14,12 @@ use histogram::Histogram as _Histogram;
 pub struct Histogram {
     /// parameters representing the resolution and the range of
     /// the histogram tracking request latencies
-    pub m: u32,
-    pub r: u32,
-    pub n: u32,
+    pub grouping_power: u8,
```

Comment:
I might end up changing this to include the entire Config when we move this to rustcommon, but I think this is fine for now.